### PR TITLE
Start bandits experiment on Nightly and Beta

### PR DESCRIPTION
**Pull request by LorenzoMinto**

### CODE REVIEW

**Review by moritzhaller — APPROVED** [review]

**Comment by LorenzoMinto** [comment]
Tested on staging. Bandit study showing correctly
```
AdServingStudy:MaximumAdNotificationsPerDay=60
BraveAds.EpsilonGreedyBanditStudy:Treatment
BraveAds.FrequencyCappingStudy:ExcludeAdIfWithinTimeWindow=0h
BraveAds.ShowCustomAdNotificationOnMacStudy:CustomAdNotificationDefaultPosition
BraveAds.UserActivityStudy:Triggers=EMPTY/Threshold=0.0/IdleTimeThreshold=5
ConversionsParametersStudy:NewDefaultPattern
EphemeralStorageStudy:Enabled
PermissionLifetimeStudy:Enabled
```

**Review by kjozwiak — APPROVED** [review]
Approved & Merged after deliberating with @brave/uplift-approvers.

**Pull request merged**
